package cz.cvut.fel.dsv.distributedComputation.exceptions;

public class AlreadyStartedException extends Exception {
}
